AppCenterDistribute/AppCenterDistribute/MSDistribute.m

```diff
@@ -1226,4 +1212,14 @@ + (void)resetSharedInstance {
   sharedInstance = nil;
 }
 
++ (void)disableAutomaticCheckForUpdates {
+  @synchronized(self) {
+    if ([MSDistribute sharedInstance].started) {
+      MSLogError([MSDistribute logTag], @"Cannot disable automatic check for updates after Distribute is started.");
+      return;
+    }
+    [MSDistribute sharedInstance].automaticCheckForUpdatesDisabled = YES;
```

Comment:
Please add instance method like before instead of calling `sharedInstance` on each line